src/server/db/base.ts

```diff
@@ -88,7 +88,7 @@ export abstract class Async {
    */
   abstract setMetadata(
     gameID: string,
```

Comment:
Ideally, we should probably rename all the `gameID` parameters in this file to `matchID`.

```suggestion
    matchID: string,
```

Reply:
I am unsure about this, because it looks like this is going to open a huge can of worms, where I'll have to go through the entire code base to determine where the word "game" is correctly used and where it isn't. I was quite content to do that only in the API / Lobby domains, as that's a smaller part and one that is not too hard to understand for me.

But maybe I'm wrong in assuming that those `db` functions are used all over the code base?

Reply:
The naming here is purely cosmetic as the ID is a string parameter (not a property in an object) so I don’t think you’d need to change the call sites, but we can also leave this as is for now and tidy up when we come to it.

Reply:
Ah, right, it's an interface file! Alright then, on it. :-)